pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -351,12 +329,9 @@ metadata:
 `
 
 	_, err := buildChangeGraph(circularDep1YAML, ctldgraph.ActualChangeOpUpsert, t)
-	if err == nil {
-		t.Fatalf("Expected graph to fail building")
-	}
-	if err.Error() != "Detected cycle while ordering changes: [job/job1 () cluster] -> [job/job2 () cluster] -> [job/job1 () cluster] (found repeated: job/job1 () cluster)" {
-		t.Fatalf("Expected to detect cycle: %s", err)
-	}
+	require.Error(t, err, "expected graph to fail building")
+
+	require.EqualError(t, err, "Detected cycle while ordering changes: [job/job1 () cluster] -> [job/job2 () cluster] -> [job/job1 () cluster] (found repeated: job/job1 () cluster)", "expected to detect cycle")
```

Comment:
Should we declare the expected error first or use a function to retrieve it to make it more readable?
Something like
```suggestion
	expectedErr := "Detected cycle while ordering changes: [job/job1 () cluster] -> [job/job2 () cluster] -> [job/job1 () cluster] (found repeated: job/job1 () cluster)"
	require.EqualError(t, err, expectedErr, "Expected to detect cycle")
```